bin/auth-api/src/routes/workspace.routes.ts

```diff
@@ -189,6 +189,10 @@ router.get("/workspaces/:workspaceId/go", async (ctx) => {
   // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
   const authUser = ctx.state.authUser!;
 
+  if (!authUser.emailVerified) {
```

Comment:
currently this is going to leave you on an ugly JSON error page - which is something we'll want to deal with eventually anyway... But we should also probably add some more checks in a few places in the auth portal and show a nice error message so they will be unlikely to get here

Reply:
We are going to have the ability to be able to trigger a verification check / re-send in the UI at some point soon. I talked ti @britmyerss about this ugly page and it's fine for now ™️